fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -1737,8 +1740,10 @@ ACTOR Future<Void> testBackupContainer(std::string url,
 			wait(testWriteSnapshotFile(range, begin, end, blockSize));
 
 			if (deterministicRandom()->random01() < .2) {
-				writes.push_back(c->writeKeyspaceSnapshotFile(
-				    snapshots.rbegin()->second, snapshotBeginEndKeys.rbegin()->second, snapshotSizes.rbegin()->second));
+				writes.push_back(c->writeKeyspaceSnapshotFile(snapshots.rbegin()->second,
+				                                              snapshotBeginEndKeys.rbegin()->second,
+				                                              snapshotSizes.rbegin()->second,
+				                                              IncludeKeyRangeMap::False));
```

Comment:
This actor is only called in unit tests so its probably fine to have this parameter as `true`

Reply:
Use `IncludeKeyRangeMap(BUGGIFY)` so that the unit test tests both modes.  This would catch, for example, where disabling the key range info makes the file not usable somehow.